README/ReleaseNotes/v626/index.md

```diff
@@ -81,6 +102,36 @@ More details at [PR #8737](https://github.com/root-project/root/pull/8737).
 - As an aid to `RDataSource` implementations with which collection sizes can be retrieved more efficiently than the full collection, `#var` can now be used as a short-hand notation for column name `R_rdf_sizeof_var`.
 - Helpers have been added to export data from `RDataFrame` to RooFit datasets. See the "RooFit Libraries" section below for more details.
 
+### Distributed RDataFrame
+The distributed RDataFrame module has been improved. Now it supports sending RDataFrame tasks to a [Dask](https://dask.org/) scheduler, for example:
+
+```python
+import ROOT
+from dask.distributed import Client
+RDataFrame = ROOT.RDF.Experimental.Distributed.Dask.RDataFrame
+
+# In a Python script the Dask client needs to be initalized in a context
+# Jupyter notebooks / Python session don't need this
+if __name__ == "__main__":
+    client = Client("SCHEDULER_ADDRESS")
+    df = RDataFrame("mytree","myfile.root", daskclient=client)
+    # Proceed as usual
+    df.Define("x","someoperation").Histo1D("x")
+```
+
+Through Dask, RDataFrame can be also scaled to a cluster of machines managed through a batch system like HTCondor or Slurm. Other notable additions and improvements include:
```

Comment:
I'd move `Through Dask, RDataFrame can...` to before the example and here have a paragraph on its own for `Other notable additions...` (otherwise it initially seems it's still related to Dask).

Lots of work for this release, congrats!